main.py

```diff
@@ -38,14 +41,25 @@ def load_config_file():
         "master_path": {master_path},
         "repository_path": {repository_path},
         "result_path": {result_path},
-        "ttl_channels": [{channel0}, {channel1}, ... ],
+        "device_db_path": {device_db_path},
+        "ttl_devices": [{ttl_device0}, {ttl_device1}, ... ],
         "dac_devices": [{dac_device0}, {dac_device1}, ... ]
       }
     """
     with open("config.json", encoding="utf-8") as config_file:
         configs.update(json.load(config_file))
 
 
+def load_device_db():
+    """Loads device DB from the device DB file."""
+    device_db_full_path = posixpath.join(configs["master_path"], configs["device_db_path"])
+    module_name = "device_db"
+    spec = importlib.util.spec_from_file_location(module_name, device_db_full_path)
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    device_db.update(module.device_db)
```

Comment:
This seems okay now, but I think we should update this to use RPC to fetch the artiq master's device DB.

Reply:
Okay! I created an issue in terms of this.